Grid/qcd/hmc/integrators/Integrator.h

```diff
@@ -480,6 +490,58 @@ class Integrator {
     return H;
   }
 
+  // LLR- implementation
+  RealD S_llr(Field& U) {
```

Comment:
This function should only be callable if the integrator is working on LLR. This could be achieved via templating.